src/editions.md

```diff
@@ -0,0 +1,39 @@
+# Editions
+
+Before we get into the details of the language itself, we need to cover the
+concept of an "edition."
+
+An "edition" of Rust consists of a particular year. There are two editions of
```

Comment:
Nit: Change `"edition"` to `*edition*`.

Not nit: Although really, we want the defining sentence to summarize the feature even if its gets expanded on later. See the comment on the core definition below for more on that.